### PR TITLE
Add stacking delegation info

### PR DESCRIPTION
**Pull request by agraebe**
This PR adds instructions for stacking delegation

### CODE REVIEW

**Comment by jeffdomke** [comment]
I keep seeing this error in staging where the whole screen is taken over by the first image on a page: 

https://blockstack-docs-git-feat-pox-delegation.blockstack.vercel.app/stacks-blockchain/stacking

Screen Shot 2020-11-24 at 11 05 52 AM

**Comment by agraebe** [comment]
I've seen this issue as well at some point. refreshing the page did the trick for me. @aulneau is there some CSS issue you noticed?

**Comment by aulneau** [comment]
Please rebase off of master and check if the error still exists. I fixed some things with my latest release.

File: src/pages/understand-stacks/integrate-stacking.md [hunk 1]
```diff
@@ -412,6 +412,16 @@ To display the unlocking time, you need to use the `firstRewardCycle` and the `l
 
 -> Coming soon: how to obtain info previous lock-ups, durations, dates, and rewards?
 
-## Notes on delegation
+## Implementing delegation
 
--> Coming soon: how to enable Stacking delegation?
+~> Right now, the tooling does not support the delegation flow. However, the delegation flow can be implemented with contract calls to the Stacking contract
+
+Stacking delegation requires you to implement a different flow. Until the provided tooling supportS this flow, it is best to implement the required contract calls directly.
```
Comment (kantai) [inline src/pages/understand-stacks/integrate-stacking.md:419]: `supportS` -> `supports`

File: src/pages/understand-stacks/integrate-stacking.md [hunk 2]
```diff
@@ -412,6 +412,16 @@ To display the unlocking time, you need to use the `firstRewardCycle` and the `l
...
+
+Below is a description of contract calls required to integrate delegation:
+
+1. Account holder call the [`delegate-stx`](/references/stacking-contract#delegate-stx) method and provide the delegator address, the amount accessible to the delegator, the address rewards should be send to, and the burn height at which the delegation relationship should be terminated
```
Comment (kantai) [inline src/pages/understand-stacks/integrate-stacking.md:423]: `call` -> `calls`
`provide` -> `provides`

File: src/pages/understand-stacks/integrate-stacking.md [hunk 3]
```diff
@@ -412,6 +412,16 @@ To display the unlocking time, you need to use the `firstRewardCycle` and the `l
...
+2. For each delegation relationship that was created, the delegator calls the [`delegator-stack-stx`](/references/stacking-contract#delegate-stack-stx) method to lock up the STX token from the account holder. This method would need to be called until the delegator locked up enough STX tokens required to participate in Stacking
```
Comment (kantai) [inline src/pages/understand-stacks/integrate-stacking.md:424]: `would need to be called` -> `must be called`?

**Review by kantai — APPROVED** [review]
LGTM!

**Pull request merged**